x/gov/handler.go

```diff
@@ -129,6 +131,52 @@ func handleMsgProposalAssignPermission(
 	}, nil
 }
 
+func handleMsgProposalSetNetworkProperty(
+	ctx sdk.Context,
+	ck keeper.Keeper,
+	msg *customgovtypes.MsgProposalSetNetworkProperty,
+) (*sdk.Result, error) {
+	isAllowed := keeper.CheckIfAllowedPermission(ctx, ck, msg.Proposer, customgovtypes.PermCreateSetNetworkPropertyProposal)
+	if !isAllowed {
+		return nil, errors.Wrap(customgovtypes.ErrNotEnoughPermissions, "PermCreateSetNetworkPropertyProposal")
```

Comment:
I have used here customgovtypes.PermCreateSetNetworkPropertyProposal.String() so it will never go out of sync.